views/partials/footer.tmpl

```diff
@@ -1,2 +1,8 @@
 {{define "footer"}}
+	<footer class="container outer footer">
+        	<div class="navbar-left"><a href="#">Disclaimer</a> | <a href="#">MyRadio</a> | <a href="#">Contact</a></div>
+        	<div class="navbar-right">
+				<p>Copyright © 1967-(Insert Current Year Here) University Radio York. Designed by the <a href="mailto:webmaster@ury.org.uk" title="Click to email the Computing team.">URY Computing Team</a></p>
```

Comment:
It'd be possible sure, but I don't think it's necessary nor even appropriate. I'm no expert on copyright law, but I don't know of anywhere that automatically updates the copyright date when the year ticks over, it's all done manually on the next update.